google/cloud/bigquery/schema.py

```diff
@@ -136,6 +150,18 @@ def fields(self):
         """
         return self._fields
 
+    @property
+    def policy_tags(self):
+        """Optional[google.cloud.bigquery.schema.PolicyTagList]: Policy tag list
+        definition for this field.
+
+        Raises:
+            ValueError:
+                If the value is not :class:`~google.cloud.bigquery.schema.PolicyTagList`
+                or :data:`None`.
```

Comment:
Since `policy_tags` is not writeable, it doesn't really raise any errors? Not even in the constructor, which simply stores the value passed to it. Let's thus delete this part.

Reply:
Done